packages/next/src/build/webpack/loaders/lightningcss-loader/src/utils.ts

```diff
@@ -0,0 +1,22 @@
+import browserslist from 'next/dist/compiled/browserslist'
+import { browserslistToTargets } from '../../../../swc/lightningcss/browserslistToTargets'
+import type { Targets } from '../../../../swc/lightningcss/targets'
+import type { ECacheKey } from './interface'
+
+let targetsCache: Record<string, Targets> = {}
+export const getTargets = (opts: {
+  default?: string | string[]
+  key: ECacheKey
+}) => {
+  const cache = targetsCache[opts.key]
+  if (cache) {
+    return cache
+  }
+  const cwd = process.cwd()
+  const result = browserslist(opts.default, {
```

Comment:
We already load browserslist earlier so you can pass the configuration from `webpack-config.ts` to the loader/plugin, which allows for skipping this

Reply:
This logic is wrong as well, it assumes `process.cwd()` is the project path which is not always the case, should use the webpack project path instead -- Note this is not relevant because for Next.js as we should use the already loaded browserslist instead.